server/repository/src/migrations/v2_06_00/add_insurance_fields_to_invoice.rs

```diff
@@ -0,0 +1,26 @@
+use crate::migrations::*;
+
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "add_insurance_fields_to_invoice"
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        sql!(
+            connection,
+            r#"
+                ALTER TABLE invoice
+                    ADD COLUMN name_insurance_join_id TEXT
+                    REFERENCES name_insurance_join (id);
+                ALTER TABLE invoice
+                    ADD COLUMN insurance_discount_amount {DOUBLE};
+                ALTER TABLE invoice
+                    ADD COLUMN insurance_discount_rate {DOUBLE};
```

Comment:
Not sure if it's worth changing but in name_insurance_join I called this `discount_percentage`
I like to explicitly say it's a percentage vs rate which could be a ratio or 0.1 for 10% for example.

Reply:
Yeah, agreed: e24245cd65